### PR TITLE
add  declare variables

### PR DESCRIPTION
**Pull request by AhmadAgus2010**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,14 @@ comment.
...
 
 ```python
-a = 'some text' # declare a string variable
-b = 123         # an integer
-c = True        # a boolean
+a = 'some text'    # declare a string variable
+b = 123            # an integer
+c = True           # a boolean
+d = "hello world"  # declare a string variable menggunakaan tanda ("") petik dua  
+e = false          # a boolean
```
Comment (imambungo) [inline README.md]: di Python, boolean diawali dengan huruf besar.  
`True` bukan `true`.  
 `False` bukan `false`.

Bakalan error kalo huruf pertamony pake huruf kecil.

**Review by imambungo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (imambungo) [reply README.md]: @AhmadAgus2010 Tolong diupdate mad

File: README.md [hunk 2]
```diff
@@ -28,7 +28,14 @@ comment.
...
+f = 3.5            # a float
+g = 2+3j           # a complex
```
Comment (imambungo) [inline README.md]: @AhmadAgus2010 mad, ini dapat dari mn? bs minta link referensiny dk? yg line 37

File: README.md [hunk 3]
```diff
@@ -28,7 +28,14 @@ comment.
...
+h = [1,2,3,4]      # a list
```
Comment (imambungo) [inline README.md]:
```suggestion
h = [1, 2, 3, 4]   # a list
```

File: README.md [hunk 4]
```diff
@@ -28,7 +28,14 @@ comment.
...
+i = ("w","o","r","l","d") # a tuple
```
Comment (imambungo) [inline README.md]:
```suggestion
i = ("w", "o", "r", "l", "d") # a tuple
```

**Review by imambungo — COMMENTED** [review]
Bagusny waktu bikin list atau tuple elemen2ny diselipkan spasi, biar rapi dan enak dilihat.

**Review by defrianafandi — COMMENTED** [review]
Good @imambungo

File: README.md [hunk 5]
```diff
@@ -28,7 +28,14 @@ comment.
 ### How to declare variables
 
 ```python
-a = 'some text' # declare a string variable
-b = 123         # an integer
-c = True        # a boolean
+a = 'some text'    # declare a string variable
```
Comment (ariefrahmansyah) [inline README.md]: Ubah dong dari `a = 'some text'` jadi `a = 'ini adalah string'`

File: README.md (hunk 1 above)
Comment (AhmadAgus2010) [reply README.md]: oke mam, baru ku update . hehe

**Review by ariefrahmansyah — CHANGES_REQUESTED** [review]
Wuoohhhh, mantabbbb @AhmadAgus2010!
Thanks for the pull request! This is really good pull request!

Btw, can you fix the merge conflict on README.md? You can refer to this guide: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github

**Review by ariefrahmansyah — APPROVED** [review]
Alright, LGTM! Thanks for your work!

**Pull request merged**